src/create-verify-archive.js

```diff
@@ -76,6 +77,49 @@ exports.createCommand = function*(argv) {
     var absOutDir = path.resolve(outDir);
 
     yield repoutil.forEachRepo(repos, function*(repo) {
+        if(isWin) {
```

Comment:
@nikhilkh I'm Ok with the change I was just curious of why the isWin, that's all.
I was not trying to blocked the merge. Merge away..
+1